procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+        2. Update `metadata` to remove anything mentioning "staging"
+        3. Update the scope to assume from the non-staging hook's role
+7. Create the role for the staging hook (e.g.: [`fenix-nightly-staging`](https://taskcluster-web.netlify.com/auth/roles/hook-id%3Aproject-mobile%2Ffenix-nightly-staging))
+    1. Go to [Roles](https://taskcluster-web.netlify.com/auth/roles)
+    2. `RoleId` should be idential to your staging hook's `HookId`, but prefixed with `hook-id:` (e.g.: `hook-id:project-mobile/fenix-nightly-staging`)
+    3. `Description` should be something like "Hook for building, signing and uploading (staging) Nightly versions of $product"
+    4. Add all the scopes you'll need to fulfill the requirements in you `.taskcluster.yml` file (when `is_mozilla_mobile_repo` is `false`)
+8. Do the same thing, but for non-staging nightlies (e.g.: [`fenix-nightly`](https://taskcluster-web.netlify.com/auth/roles/hook-id%3Aproject-mobile%2Ffenix-nightly))
+    1. Remove mentions of "staging" from each field
+    2. Make sure that all scopes are provided to fulfill requirements in `.taskcluster.yml` (when `is_mozilla_mobile_repo` is `true`)
+9. Update `scriptworker`, `pushapk` and `mozapkpublisher`
+    1. View the example changes for `fenix` and replicate them for the product you're automating: [example `scriptworker` changes](https://github.com/mozilla-releng/scriptworker/pull/298), [example `pushapk` changes](https://github.com/mozilla-releng/pushapkscript/pull/69) and [example `mozapkpublisher` changes](https://github.com/mozilla-releng/mozapkpublisher/pull/149)
+    2. Once these changes are CR'd and merged, publish a new minor version of each of these packages
+        1. Make PRs: [example `pushapkscript` release PR](https://github.com/mozilla-releng/pushapkscript/pull/70)
+        2. Once these release PRs are CR'd and merged, publish each package:
+            1. `git checkout master`, make sure you're on the "Merge pull request" commit for your PR
+            2. `git tag $version`, e.g.: `git tag 0.11.0`
+            3. `git push --tags && git push upstream --tags` (assuming that the `origin` remote is for your fork, and `upstream` is the `mozilla-releng` repo)
+            4. Ensure you're in the Python virtual env for your package (One approach is to share a single virtual env between all scriptworker repos)
+            5. `rm -rf dist && python setup.py sdist bdist_wheel` build the package
+            6. Publish to `Pypi`:
+                1. `gpg --list-secret-keys --keyid-format long` to get your GPG identity (it's the bit after "sec   rsaxxxx/"). An example GPG identity would be `5F2B4756ED873D23`
+                2. `twine upload --sign --identity $identity dist/*` to upload to Pypi (you may need to `pip install twine` first)
+
+10. Update configuration in [`build-puppet`](https://github.com/mozilla-releng/build-puppet/)
+    1. Locate signing secrets (there's six different items you'll add to `hiera`)
```

Comment:
Nit: should we expand for clarity what those six items are? Even though you refer to them immediately below, it might be useful to clarify this even more.